.travis.yml

```diff
@@ -15,7 +15,7 @@ before_install:
   - touch $HOME/.android/repositories.cfg
   - yes | sdkmanager --update
 install:
-  - echo y | sdkmanager "ndk;21.0.6113669" >/dev/null
+  - echo y | sdkmanager "ndk;21.1.6352462" >/dev/null
```

Comment:
Do we need a test matrix to make sure we can still build against older NDKs?

Reply:
@bruno-garcia sentry-native already does it https://github.com/getsentry/sentry-native/blob/master/azure-pipelines.yml#L47

Reply:
TIL: Nice!! Tks